core.coffee

```diff
@@ -31,6 +31,10 @@
 		'expandFilter'
 		'`$filter: a: b: ...` is deprecated, please use `$filter: a: $any: { $alias: "x", $expr: x: b: ... }` instead.'
 	)
+	addDeprecated(
+		'implicitOptions'
+		'options: filter: ...` is deprecated, please use `options: $filter: ...` instead.'
```

Comment:
This isn't only filter right, it's also select/expand/etc? Right now this message isn't too clear on that - if saw this, I'd go looking for the offending filter specifically, rather than checking all my options.

Reply:
Yeah, it's all of them, do you have any ideas on clear/generic wording?

Reply:
Maybe something like:

> Options without $ prefixes (e.g. `options: filter: ...`) are deprecated, please use `options: $filter: ...` instead.`

Reply:
I've updated it, let me know what you think